### PR TITLE
Permision to merge to dev branch.

### PR DESCRIPTION
**Pull request by adex001**
This PR contains the updated question route.

All previous functionalities remain valid!

image

### CODE REVIEW

File: src/components/questions/questionController.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        message: 'An internal server error has occured!',
+        err,
+      });
+    }
+  }
+  async dailyQuestions(req, res, next) {
+    try {
+      const newDailyQuestion = await dailyQuestions.create(req.body);
+      return res.status(201).json({
+        dailyQuestion: newDailyQuestion,
```
[the remark this answers is not in the document]
Comment (adex001) [reply src/components/questions/questionController.js]: ok! I will get back to it straight away!

[the remark this answers is not in the document]
Comment (timolinn) [reply src/components/questions/questionController.js]: Seen. @adex001 can you fix the merge conflict? do you know how to?

[the remark this answers is not in the document]
Comment (timolinn) [reply src/components/questions/questionController.js]: @adex001 please sync your repo and make a new PR.

[the remark this answers is not in the document]
Comment (adex001) [reply src/components/questions/questionController.js]: yeah!! the newest merge resulted into a conflict. I will update and remove redundant files.

**Pull request closed without merging**